src/index.html

```diff
@@ -11,7 +11,169 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="data_block">
+        <legend class="title_personal">Personal information</legend>
+        <label class="block">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="block">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="block">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            max="100"
+            min="1"
+            required
+          >
+        </label>
+
+        <label class="block">
+          Full date of birth:
+          <input
+            type="date"
+            name="date_of_birth"
+            required
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="сonsent"
+            required
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset class="data_block">
```

Comment:
```suggestion
      </fieldset>

      <fieldset class="data_block">
```

Add only one empty line between multiple sibling blocks

Check all such cases

Reply:
https://dmitryvolchokfe.github.io/HTML_form/report/html_report/
https://dmitryvolchokfe.github.io/HTML_form/
Посилання котрі працюють